frontend/src/utils/date.ts

```diff
@@ -7,14 +7,16 @@ export const formatFuzzyDate = (date: FuzzyDateInput) => {
   return date.date.slice(0, 4);
 };
 
-export const formatDateTime = (dateTime: Date | string) => {
+export const formatDateTime = (dateTime: Date | string, utc = false) => {
   const date = dateTime instanceof Date ? dateTime : new Date(dateTime);
   return `${date.toLocaleString("en-us", {
     month: "short",
     year: "numeric",
     day: "numeric",
     timeZone: "UTC",
```

Comment:
I think you may need to add the same here, because the date in UTC might not match the date in local time zone.

Reply:
Good catch, fixed, thanks!